SafeExamBrowser.SystemComponents/Registry/Registry.cs

```diff
@@ -86,6 +89,59 @@ public bool TryRead(string key, string name, out object value)
 			return success;
 		}
 
+		public bool TryGetNames(string key, out IEnumerable<string> names)
+		{
+			names = default;
+
+			RegistryKey keyObj;
+			if (!TryOpenKey(key, out keyObj))
+				return false;
+
+			bool success = true;
```

Comment:
Whenever possible and meaningful, it would be great to use `var` for consistency and readability reasons with respect to the overall code base.